provisioning/baremetal_pod_test.go

```diff
@@ -357,13 +357,11 @@ func TestNewMetal3Containers(t *testing.T) {
 				withEnv(
 					containers["metal3-httpd"],
 					envWithValue("PROVISIONING_INTERFACE", ""),
-					envWithFieldValue("PROVISIONING_IP", "status.hostIP"),
```

Comment:
I'd like to see one more test case where the provisioning IP is not set, and we fall back to status.hostIP

Reply:
Isn't that the test just below, L374? `DisabledSpecWithoutProvisioningIP`?